version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.0.1"
+	Version = "5.5"
```

Comment:
How will this be populated other then a manual bump?

Reply:
Makefile got a target version-update where i am doing the below  :

version-update:
	sed -i 's/"[0-9]*\.[0-9]*\.[0-9]*"/\"${LOGGING_VERSION}\"/g' version/version.go
bin/cluster-logging-operator: version-update force
	@oc -n openshift-cluster-version scale deployment/cluster-version-operator --replicas=$(REPLICAS)

Reply:
This is not a sustainable process.  Please review a running container and see if there is maybe an environment variable from which we can retrieve this info.  There are many variables that are injected into the container at runtime

Reply:
Example of where to find the information.  We will have to fix our CI to bake this into the `version.go`
```
 containers:
    - resources: {}
      terminationMessagePath: /dev/termination-log
      name: cluster-logging-operator
      command:
        - cluster-logging-operator
      env:
        - name: OPERATOR_CONDITION_NAME
          value: cluster-logging.5.4.0-138
```